src/pages/workspace/WorkspaceInitialPage.js

```diff
@@ -121,7 +122,7 @@ class WorkspaceInitialPage extends React.Component {
             {
                 translationKey: 'workspace.common.bankAccount',
                 icon: Expensicons.Bank,
-                action: () => Navigation.navigate(ROUTES.getWorkspaceBankAccountRoute(policy.id)),
+                action: () => WorkSpaceUtils.getShouldShowBankOrNonAccountPage(this.props, ROUTES.getWorkspaceBankAccountRoute(policy.id), ROUTES.getBankAccountRoute()),
```

Comment:
Is this call matching with the function signature?

Reply:
Changes applied.